### PR TITLE
Switch to ToolforgeBundle

### PR DESCRIPTION
**Pull request by samwilson**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,21 +25,26 @@
...
         "symfony/swiftmailer-bundle": "^3.0",
         "symfony/symfony": "^4.0",
-        "twig/twig": "^2.0"
+        "twig/twig": "^2.0",
+        "wikimedia/toolforge-bundle": "^0.3"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/wikimedia/ToolforgeBundle.git"
```
Comment (MusikAnimal) [inline composer.json]: Do we still need this?

Comment (samwilson) [reply composer.json]: Nope, not now it's on packagist; I'll update now.

**Comment by samwilson** [comment]
Oops! I was testing a thing and removed it from webpack.config.js. Added now; see what you think.

**Comment by MusikAnimal** [comment]
$.i18n works great now!

I am not able to get `logged_in_user` to work. I tried putting
```yaml
toolforge:
    oauth:
        logged_in_user: 'MusikAnimal (WMF)'
```
in parameters.yml and overriding the `'%app.logged_in_user%'` directly in config.yml. Yet, it wants me to login on Meta. Am I doing it correctly? The goal here would be to have a way to specify the logged in user without changing code in version control.

Once we figure this out, we should update the [README](https://github.com/wikimedia/grantmetrics#installation-for-development).

**Comment by samwilson** [comment]
Hm, strange: it's working for me just with `app.logged_in_user: Samwilson` in parameters.yml. I'll dig a bit more. Could it be a cache issue?

**Comment by samwilson** [comment]
I had a weird local config. It was indeed failing. Can you try again now?

**Comment by MusikAnimal** [comment]
Works great! Merging

**Pull request merged**